project/application/src/components/PlaningUserBlock/style.scss

```diff
@@ -236,6 +246,28 @@
           border-right: 0;
         }
       }
+
+      .cell-isWeek {
+        box-sizing: content-box;
+        position: relative;
+        width: 158px;
+        min-width: 158px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 20px;
```

Comment:
you need to understand what will happen during overflow, using a fixed height is dangerous